app/templates/main/search-transferring-body.html

```diff
@@ -22,6 +22,7 @@
     "Record status (closed)": "closure_type-asc",
     "Record status (open)": "closure_type-desc"
 } %}
+{% set anchor_fields = {"file_name": "file_id"} %}
```

Comment:
I think we can get rid of this dict and simplify your logic below to

`if first_result_key == "file_name`


`<a href="{{ url_for('main.record', record_id=record['_source']["file_id"]) }}">{{ first_result_value }}</a>`